packages/cmrjs/cmr-utils.js

```diff
@@ -18,8 +18,9 @@ const {
 } = require('@cumulus/common');
 const { DefaultProvider } = require('@cumulus/common/key-pair-provider');
 const { deprecate, omit } = require('@cumulus/common/util');
+const { getLaunchpadToken } = require('@cumulus/common/launchpad');
 
-const { CMR } = require('./cmr');
+const { CMR } = require('@cumulus/cmr-client');
```

Comment:
I think this is okay? Since it will be using the new version. Just want to make sure there's nothing in the old one we still need.

Reply:
I think we should remove reference to the old version in our packages.
https://bugs.earthdata.nasa.gov/browse/CUMULUS-1437